pkg/blob/controllerserver.go

```diff
@@ -691,3 +807,58 @@ func parseDays(dayStr string) (int32, error) {
 
 	return int32(days), nil
 }
+
+func GenerateSASToken(accountName, accountKey string) (string, error) {
+	credential, err := azblob.NewSharedKeyCredential(accountName, accountKey)
+	if err != nil {
+		return "", status.Errorf(codes.Internal, fmt.Sprintf("failed to generate sas token in creating new shared key credential, accountName: %s, accountKey: %s, err: %s", accountName, accountKey, err.Error()))
+	}
+	serviceClient, err := service.NewClientWithSharedKeyCredential(fmt.Sprintf("https://%s.blob.core.windows.net/", accountName), credential, nil)
```

Comment:
don't hard code `blob.core.windows.net`, it won't work on other sovereign clouds

Reply:
done